routes/api/user/index.js

```diff
@@ -70,4 +71,59 @@ router.post('/:user_id/role', (req, res, next) => {
   .catch(next);
 });
 
+/**
+ * expects 2 fields in the body of the request
+ * 1) the token that was in the url of the email link {String}
+ * 2) the new password {String}
+ */
+router.post('/update-password', (req, res, next) => {
+  const {token, password} = req.body;
+
+  User.verifyToken(token)
+    .then(user => {
+      return User.changePassword(user.id, password);
+    })
+    .then(() => {
+      res.status(204).end();
+    })
+    .catch(error => {
+      console.error(error);
+      res.status(401).send('Not Authorized');
+    });
+});
+
+/**
+ * this endpoint takes an email (username) and checks if it belongs to a User account
+ * if it does, create a JWT and send an email
+ */
+router.post('/request-password-reset', (req, res, next) => {
+  const {email} = req.body;
+
+  if (!email) {
+    return next();
+  }
+
+  User
+    .createJWT(email)
+    .then(token => {
+      const options = {
+        subject: 'Password Reset Requested - Talk',
+        from: 'coralcore@mozillafoundation.org',
+        to: email,
+        html: `<p>We received a request to reset your password. If you did not request this change, you can ignore this email.
```

Comment:
Also, associating a language with the template filename and using the i18n language name will give us the ability to send multilingual emails!

For example, this email could be:

/views/emails/reset_password_EN.ejs